Makefile

```diff
@@ -6,6 +6,8 @@ multiboot2_binary = target/$(MULTIBOOT2_TARGET)/$(BUILD_TYPE)/mythril_multiboot2
 mythril_src = $(shell find . -type f -name '*.rs' -or -name '*.S' -or -name '*.ld' \
 	                   -name 'Cargo.toml')
 seabios = seabios/out/bios.bin
+git_hooks_src = $(wildcard .mythril_githooks/*)
+git_hooks = $(subst .mythril_githooks,.git/hooks,$(git_hooks_src))
```

Comment:
should we install the hook to `pre-commit` instead of `pre-commit-cargo-fmt` so it is actually run? or do we want to leave that to the user in case they have their own `pre-commit` hook?

Reply:
Nice catch, I swear it was anything starting with pre-commit, but you're right; only that exact name is run.